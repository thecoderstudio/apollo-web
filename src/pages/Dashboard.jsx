import AgentList from '../components/agent-list/AgentList'
import ContentContainer from '../components/Content'
import React from 'react';
import styled from 'styled-components'


const Content = styled(ContentContainer)`
  grid-template-columns: [agent-listing] 1fr 1fr;
  grid-template-rows: 1fr;  
`;

export default function Dashboard(props) {
  return (
    <Content>
      <AgentList />
    </Content>
  )
}