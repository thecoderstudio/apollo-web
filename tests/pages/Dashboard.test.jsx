import configureStore from 'redux-mock-store';
import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from '../../src/pages/Dashboard';
import { Provider } from 'react-redux';

describe('dashboard', () => {
  it("renders correctly", () => {

    let store = configureStore([])
    store.dispatch = jest.fn()

    const tree = renderer.create(
      <Provider store={store()} >
        <Dashboard />
      </Provider >
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
