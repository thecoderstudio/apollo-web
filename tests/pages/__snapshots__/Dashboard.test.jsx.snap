// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard renders correctly 1`] = `
.c1 {
  display: grid;
  grid-template-rows: [title] 50px [list] 1fr;
  max-width: 100%;
  min-width: 0;
  border-radius: 8px;
  padding: 20px;
}

.c2 {
  grid-row: title;
  width: 100%;
  min-width: 0;
}

.c3 {
  width: 100%;
  min-width: 0;
  grid-row: list;
  list-style: none;
  padding-left: 0;
}

.c0 {
  display: grid;
  grid-template-columns: [agent-listing] 1fr 1fr;
  grid-template-rows: 1fr;
  margin: 25px;
}

@media (max-width:47.9375em) {
  .c0 {
    grid-template-columns: [agent-listing] 1fr;
  }
}

<div>
  <div
    className="c0"
  >
    <div
      className="c1"
    >
      <h2
        className="c2"
      >
        Agents
      </h2>
      <ul
        className="c3"
      />
    </div>
  </div>
</div>
`;
